py27googledriveconnect/googledriveutils.py

```diff
@@ -0,0 +1,164 @@
+"""
+Written By: Kathryn Cogert
+For: Winkler Lab/CSE599 Winter Quarter 2016
+Purpose: Write and read from a google doc
+
+To Do:
++Add appending data to existing files functions (write now everything just overwrites)
++Add exporting from cRIO portion
+
+
+Note: Had to copy of local copy of PyDrive because it has some issues that make it incompatible with the latest
+version of the googleAPI.  This copy is in it's own directory and
+Specifically, it forces a / at the end of the redirect URI.  Google does not allow a slash at the end of those URLS
+so in order to use the package, I had to copy it and remove that / from the end of the procedurally generated URL.
+"""
+
+from reactorpydrive.auth import GoogleAuth
+from reactorpydrive.drive import GoogleDrive
+
+
+
+"""
+Define Custom Errors
+"""
+class BaseError(Exception):
+    """Base error for google drive manipulation and navigation errors."""
+
+
+class InvalidDir(BaseError):
+    """Error for specifying a directory ID that doesn't exist"""
+
+
+class NotMatching(BaseError):
+    """Error for specifying a file name that doesn't exist"""
+
+
+class NoFolders(BaseError):
+    """Error for specifying a directory to find folders in that has no folders in it"""
+
+
+class NoSuchReactor(BaseError):
+    """Error for specifying a reactor number to find the directory for"""
+
+
+"""
+Autheticate the connection to google drive
+This will open a web browser and prompt for user permission.
+"""
+gauth = GoogleAuth()
+gauth.LocalWebserverAuth()  # Creates local webserver and auto handles authentication
+drive = GoogleDrive(gauth) # Create GoogleDrive instance with authenticated GoogleAuth instance
```

Comment:
Wouldn't it be more compliant with pep8 to comment something a line before, not on the same line after the sentence? Just some thoughts..